Twig/DataTableExtension.php

```diff
@@ -9,28 +9,26 @@
 
 class DataTableExtension extends AbstractExtension
 {
-    /**
-     * @var TwigRenderer
-     */
-    protected $renderer;
+    protected TwigRenderer $renderer;
 
     public function __construct(TwigRenderer $renderer)
     {
         $this->renderer = $renderer;
     }
+
     /**
      * {@inheritdoc}
      */
-    public function getFunctions()
+    public function getFunctions(): array
```

Comment:
be careful that this new signature matches the signature of the twig class AbstractExtension

Reply:
Corrected. Done.